src/state_machine_tests.zig

```diff
@@ -128,7 +120,7 @@ pub const TestContext = struct {
             ctx.time_sim.time(),
             &ctx.grid,
             .{
-                .batch_size_limit = message_body_size_max,
+                .batch_size_limit = constants.message_body_size_max,
```

Comment:
This is runtime limit, right? So we _don't_ have to change it here, and can keep it as is, right? 

Are we changing it intentionally then? (smaller limit (so, pre-PR) potentially feels more interesting to me here, as it'll also check runtime limiting logic)